templates/user/dashboard/dashboard.tmpl

```diff
@@ -8,7 +8,24 @@
 				{{template "user/dashboard/feeds" .}}
 			</div>
 			<div id="dashboard-repo-search" class="six wide column">
-				<repo-search :search-limit="searchLimit" :suburl="suburl" :uid="uid"><i class="fa fa-spinner fa-spin"></i></repo-search>
+				<repo-search
+					:search-limit="searchLimit"
+					:suburl="suburl"
+					:uid="uid"
+					:more-repos-link="'{{.ContextUser.HomeLink}}'"
+					{{if not .ContextUser.IsOrganization}}
+					:orgs="[
+						{{range .ContextUser.Orgs}}
+						{name: '{{.Name}}', num_repos: '{{.NumRepos}}'},
+						{{end}}
+					]"
+					:is-org="false"
+					:orgs-total="{{.ContextUser.GetOrganizationCount}}"
```

Comment:
Yes, it is. It's same as `reposTotal`. Now gitea loads everything at once, but if we add pagination (and maybe search) in the future, we don't need to care about (I think repositories can and should be preloaded this way too, instead of making init search).